main.js

```diff
@@ -0,0 +1,255 @@
+            } else {
+                divKey.innerHTML = `
+                        <span class="key-value eng">${keysEn[i][k]}</span>
+                        `
+            }
+
+            divRow.append(divKey);
+        }
+        keyboard.append(divRow);
+    }
+    const text = document.createElement('p');
+
+    text.innerHTML = `Смена языка произвотится нажатием Ctrl + Alt.  Задание выполнено в Windows10.`;
+
+    container.append(output, keyboard, text)
+    document.body.append(container);
+
+
+
+
+    /*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Ловим нажатие на клавиши  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+
+
+
+    document.addEventListener('keydown', function (event) {
+        const allKeys = [...document.querySelectorAll('.key')];
+
+        event.preventDefault();
+
+
+
+        if (event.code === 'ShiftLeft' || event.code === 'ShiftRight') {
+
+
+
+            allKeys.forEach((key) => {
+                if (!key.classList.contains('spec-key')) {
+                    key.querySelectorAll('span').forEach((span) => {
+                        if (!span.classList.contains('hidden')) {
+                            span.innerHTML = span.innerHTML.toUpperCase();
+                        }
+                    })
+                }
+            })
+        }
+
+        allKeys.forEach((key) => {
+            if (event.code === key.dataset.keyProp) {
+                key.classList.add('active');
+            }
+        });
+
+        if (event.ctrlKey && event.altKey) {
+            allKeys.forEach((key) => {
+                if (!key.dataset.keyProp.includes('Digit') && !key.classList.contains('spec-key')) {
+                    key.querySelectorAll('span').forEach((span) => {
+                        span.classList.toggle('hidden');
+                    })
+                }
+            })
+        }
+
+        printLetter();
+
+
+
+        output.focus();
+        output.selectionStart = output.value.length;
+
+
+    })
+
+    document.addEventListener('keyup', function (event) {
+        const allKeys = [...document.querySelectorAll('.key')];
+
+        if (event.code === 'ShiftLeft' || event.code === 'ShiftRight') {
+            allKeys.forEach((key) => {
+                if (!key.classList.contains('spec-key')) {
+                    key.querySelectorAll('span').forEach((span) => {
+                        if (!span.classList.contains('hidden')) {
+                            span.innerHTML = span.innerHTML.toLowerCase();
+                        }
+                    })
+                }
+            })
+        }
+
+        allKeys.forEach((key) => {
+            if (event.code === key.dataset.keyProp) {
+                key.classList.remove('active');
+            }
+        })
+
+
+    })
+
+    function printLetter() {
+        const allKeys = document.querySelectorAll('.key');
+
+
+        allKeys.forEach((key) => {
+            if (key.classList.contains('active') && key.classList.contains('enter')) {
```

Comment:
you can keep implementation, but move this `if` condition ouside `key.classList.contains('active')`, so if it's not active programm will not check everything else.